src/ingress_definition.py

```diff
@@ -316,16 +316,54 @@ def proxy_connect_timeout(self) -> str:
 
     @property
     def rewrite_enabled(self) -> bool:
-        """Return whether rewriting should be enabled from config or relation."""
-        value = self._get_config_or_relation_data("rewrite-enabled", True)
+        """
+        Return whether rewriting should be enabled from config, per-app ingress
+        relation (strip_prefix), or nginx-route relation, in that order.
```

Comment:
Hi @weiiwang01 wanted to clarify on this ordering priority: config -> ingress relation -> nginx-route relation when defining the value of `rewrite_enabled`, `rewrite_target`, and `path_routes`.